From 4fb6dd246753121e92c93b414f588cdc7d1fb69d Mon Sep 17 00:00:00 2001
From: Ozzyboshi <gun10137@gmail.com>
Date: Sat, 15 Aug 2026 01:05:30 +0200
Subject: [PATCH] adjust copperlist color for resistance image

---
 dec/6.s | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/dec/6.s b/dec/6.s
index c30565d..aec64bf 100644
--- a/dec/6.s
+++ b/dec/6.s
@@ -588,11 +588,11 @@ TITLELOGO_PTR_4_H:
 	DC.W	$00ee
 TITLELOGO_PTR_4_L:
 	DC.W	$5068
-	DC.L	$01800000 ;485c4
-	DC.L	$01820fff,$0184081f,$0186095f,$01880b8f ;485d4
-	DC.L	$018a0dcf,$018c0145,$018e08fc,$01900032 ;485e4
-	DC.L	$01920053,$01940064,$01960085,$019800a6 ;485f4
-	DC.L	$019a01c7,$019c01d8,$019e01f9,$6c11fffe ;48604
+	DC.L	$01800cc9 ;485c4
+	DC.L	$01820ba8,$01840997,$01860986,$01880886 ;485d4
+	DC.L	$018a0875,$018c0774,$018e0864,$01900665 ;485e4
+	DC.L	$0192654,$01940441,$01960444,$01980433 ;485f4
+	DC.L	$019a0220,$019c0111,$019e0000,$6c11fffe ;48604
 
 BPLPTR_UPPER_BANNER:
 	DC.L	$01000200,$6e11fffe,$01004200,$00e00004 ;48614  barra alta: WAIT $6E (era $6F)
-- 
2.43.0

